dali/test/python/test_operator_nemo_asr_reader.py

```diff
@@ -0,0 +1,158 @@
+from nvidia.dali.pipeline import Pipeline
+import nvidia.dali.ops as ops
+import nvidia.dali.types as types
+import scipy.io.wavfile
+import numpy as np
+import math
+import json
+import librosa
+import tempfile
+import os
+
+# generate sinewaves with given frequencies,
+# add Hann envelope and store in channel-last layout
+def generate_waveforms(length, frequencies):
```

Comment:
This is common for audio decoder test, maybe we can extract it to utis?

Reply:
will do